status/private/contracts.nim

```diff
@@ -0,0 +1,239 @@
+  ): TokenData
+
+contract(StickerPackContract):
+  # uncomment all commented methods once
+  # https://github.com/status-im/nim-web3/pull/39 is merged
+  proc approve(
+    to: Address,
+    tokenId: UInt256
+  )
+  proc balanceOf(
+    tokenHolder: Address
+  ): UInt256 {.view.}
+  proc getApproved(
+    tokenId: UInt256
+  ): Address {.view.}
+  proc name(): string {.view.}
+  proc ownerOf(
+    tokenId: UInt256
+  ): Address {.view.}
+  proc safeTransferFrom(
+    `from`: Address,
+    to: Address,
+    tokenId: UInt256
+  )
+  proc symbol(): string {.view.}
+  proc tokenByIndex(index: UInt256): UInt256 {.view.}
+  proc tokenOfOwnerByIndex(
+    owner: Address,
+    index: UInt256
+  ): UInt256 {.view.}
+  proc tokenPackId(
+    tokenId: UInt256
+  ): UInt256 {.view.}
+  proc tokenURI(
+    tokenId: UInt256
+  ): string {.view.}
+  proc totalSupply(): UInt256 {.view.}
+  proc setApprovalForAll(
+    owner: Address,
+    operator: Address
+  ): Bool {.view.}
+  proc isApprovedForAll(
+    owner: Address,
+    operator: Address
+  ): Bool {.view.}
+  proc transferFrom(
+    `from`: Address,
+    to: Address,
+    tokenId: UInt256
+  )
+
+# Must export all generated contracts types and methods. Can remove this entire
+# export block if https://github.com/status-im/nim-web3/pull/40 is merged.
+export
+  Erc20Contract, balanceOf,
+  MiniMeContract,
+  StickersContract, getPackData, packCount,
+  StickerMarketContract, buyToken, getTokenData,
+  StickerPackContract, approve, balanceOf, getApproved, name, ownerOf,
+    safeTransferFrom, symbol, tokenByIndex, tokenOfOwnerByIndex, tokenPackId,
+    tokenURI, totalSupply, transferFrom
+
+const
+  STICKERS_CONTRACT_ADDRESSES* = {
+    NetworkId.Mainnet.int:
+      Address.fromHex("0x0577215622f43a39f4bc9640806dfea9b10d2a36"),
+    NetworkId.Rinkeby.int:
+      Address.fromHex("0x8cc272396Be7583c65BEe82CD7b743c69A87287D"),
+    NetworkId.Ropsten.int:
+      Address.fromHex("0x8cc272396Be7583c65BEe82CD7b743c69A87287D"),
+    NetworkId.Goerli.int:
+      Address.fromHex("0x8cc272396Be7583c65BEe82CD7b743c69A87287D")
+  }.toTable
+  STICKERMARKET_CONTRACT_ADDRESSES* = {
+    NetworkId.Mainnet.int:
+      Address.fromHex("0x12824271339304d3a9f7e096e62a2a7e73b4a7e7"),
+    NetworkId.Rinkeby.int:
+      Address.fromHex("0x6CC7274aF9cE9572d22DFD8545Fb8c9C9Bcb48AD"),
+    NetworkId.Ropsten.int:
+      Address.fromHex("0x6CC7274aF9cE9572d22DFD8545Fb8c9C9Bcb48AD"),
+    NetworkId.Goerli.int:
+      Address.fromHex("0x6CC7274aF9cE9572d22DFD8545Fb8c9C9Bcb48AD")
+  }.toTable
+  STICKERPACK_CONTRACT_ADDRESSES* = {
+    NetworkId.Mainnet.int:
+      Address.fromHex("0x110101156e8F0743948B2A61aFcf3994A8Fb172e"),
+    NetworkId.Rinkeby.int:
+      Address.fromHex("0xf852198D0385c4B871E0B91804ecd47C6bA97351"),
+    NetworkId.Ropsten.int:
+      Address.fromHex("0xf852198D0385c4B871E0B91804ecd47C6bA97351"),
+    NetworkId.Goerli.int:
+      Address.fromHex("0xf852198D0385c4B871E0B91804ecd47C6bA97351")
+  }.toTable
```

Comment:
These contract addresses won't be changing often, if ever, and didn't feel the need to create DB tables for them. There is definitely no need for CRUD support or custom contracts for stickers at this point in time.